src/Components/CreateImageWizard/formComponents/ReviewStepTables.js

```diff
@@ -20,18 +20,23 @@ const RepoName = ({ repoUrl }) => {
     url: repoUrl,
   });
 
+  const errorLoading = () => {
+    return (
+      <Alert
+        variant="danger"
+        isInline
+        isPlain
+        title="Error loading repository name"
+      />
+    );
+  };
+
   return (
     <>
-      {isSuccess && <p>{data.data?.[0].name}</p>}
+      {isSuccess && data.data?.[0]?.name && <p>{data.data?.[0].name}</p>}
```

Comment:
The succeeding blocs are a bit confusing for me to understand sorry.
Can you comment each of these lines please ?